### ISSUE TITLE
Facepile reads "1 of 1" for each persona on screenreader

### ISSUE THREAD

**Issue by mowei8** [OfficeDev/office-ui-fabric-react#6758]
### Bug Report 
 
 
- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No) No
 
Requested priority: (Blocking, High, Normal, Low) High
 
Products/sites affected: (if applicable)
 
#### Actual behavior: When you tab to a persona in the Facepile list, the screen reader will always repeat "1 of 1" after saying the name. For example in the image below, there are two users in the list, but it will read "Monica Wei, 1 of 1, not selected". 
![image](https://user-images.githubusercontent.com/39922684/47186322-0f400e00-d2e5-11e8-95fe-35b94608cd0a.png)

**Comment by aneeshack4** [comment]
Just want to update this issue with notes on everything I've tried so far to fix this: 
role="list" on the <ul> element and role="listitem" on the <li> element in Facepile.base.tsx
role="listbox" on the <ul> and role="option" and aria-posinset={index+1} and aria-setsize={personas.length}. I've been cross referencing my implementation against examples like the ones on http://www.maxability.co.in/demos/aria-posinset.html and I don't see any difference. On top of that, Narrator doesn't run well on these examples to begin with so I'm going to reach out to the Narrator team and speak with them to see if there's something wrong on their end or if there's something I'm missing.

**Comment by aneeshack4** [comment]
Spoke with the Narrator team and learned that they recently made a change get "n of m" added to the end of every element's announcement if that element supports posinset and sizeofset. This change hasn't been released publicly yet though so once that happens, this issue should be checked again. Closing for now.
